javascript/core/dummy-workout/if-else-shortcut-conditional-operator.md

```diff
@@ -66,7 +66,7 @@ var animal = 'lion';
 var run = (animal === 'lion')? true:???;
 ```
 
-* 'false'
+* false
```

Comment:
Can we also format this code to make it a bit more readable? Mainly:

```js
var run = (animal === 'lion')? true:???;
```

to

```js
var run = (animal === 'lion') ? true : ???;
```

Reply:
In general, you can just paste JavaScript snippet into https://prettier.io/playground/ to get a nicely formatted output (until we implement a better, generalized system for this)

Reply:
Can't add the space because it would make the code go out of the <44 character screen limit